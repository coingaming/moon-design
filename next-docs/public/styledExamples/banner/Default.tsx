import { Banner } from '@heathmont/moon-core';

const Example = () => (
  <Banner title="Title here" description="Lorem ipsum dolor..." />
);

export default Example;
