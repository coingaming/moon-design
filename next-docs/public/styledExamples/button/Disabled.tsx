import { Button } from '@heathmont/moon-core';

const Example = () => <Button disabled>Disabled</Button>;

export default Example;
