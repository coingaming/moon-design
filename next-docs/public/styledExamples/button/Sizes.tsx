import { Button } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Button size="xsmall">xSmall</Button>
    <Button size="small">Small</Button>
    <Button>Medium is default</Button>
    <Button size="large">Large</Button>
    <Button size="xlarge">xLarge</Button>
  </>
);

export default Example;
