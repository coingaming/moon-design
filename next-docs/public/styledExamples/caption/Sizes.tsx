import { Caption } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Caption>Default font-size is 10px</Caption>
    <Caption size={9}>Size is 9px</Caption>
  </>
);

export default Example;
