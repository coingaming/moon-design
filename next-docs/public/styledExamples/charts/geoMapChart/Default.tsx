// @ts-nocheck
import React from 'react';
import { Filter, GeoMapChart } from '@heathmont/moon-charts';
import { SingleItemSelect } from '@heathmont/moon-components';

const Example = () => {
  const data = [
    { label: 'Estonia', code: 'EST', value: 120 },
    { label: 'Russia', code: 'RUS', value: 100 },
    { label: 'Belarus', code: 'BLR', value: 80 },
    { label: 'Argentina', code: 'ARG', value: 60 },
    { label: 'USA', code: 'USA', value: 40 },
  ];

  const filter = (
    <Filter title="by GGR">
      {() => (
        <SingleItemSelect
          onChange={() => {}}
          value="ggr"
          options={[
            { label: 'Active players', value: 'actives' },
            { label: 'GGR', value: 'ggr' },
            { label: 'Bets', value: 'bets' },
          ]}
        />
      )}
    </Filter>
  );

  return (
    <GeoMapChart
      title="Top Countries"
      data={data}
      filter={filter}
      hasUpdates={true}
      onUpdate={() => {}}
      onShare={() => {}}
      onExpand={() => {}}
    />
  );
};

export default Example;
