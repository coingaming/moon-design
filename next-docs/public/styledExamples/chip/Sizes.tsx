import { Chip } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Chip size="small">Small</Chip>
    <Chip>Medium is Default</Chip>
  </>
);

export default Example;
