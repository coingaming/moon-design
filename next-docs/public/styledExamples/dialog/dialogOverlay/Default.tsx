import React from 'react';
import { DialogOverlay } from '@heathmont/moon-components';
import { Button } from '@heathmont/moon-core';

const Example = () => {
  const [showDialog, setShowDialog] = React.useState(false);

  return (
    <>
      <Button variant="tertiary" onClick={() => setShowDialog(true)}>
        Open Dialog
      </Button>
      <DialogOverlay
        isOpen={showDialog}
        onDismiss={() => setShowDialog(false)}
      />
    </>
  );
};

export default Example;
