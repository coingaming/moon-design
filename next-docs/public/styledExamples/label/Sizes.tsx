import React from 'react';
import { Label } from '@heathmont/moon-core';

const Example = () => (
  <>
    <Label size="twoxsmall">Twoxsmall</Label>
    <Label>Xsmall</Label>
    <Label size="twoxsmall" isUppercase={false}>
      Twoxsmall
    </Label>
    <Label isUppercase={false}>Xsmall</Label>
  </>
);

export default Example;
