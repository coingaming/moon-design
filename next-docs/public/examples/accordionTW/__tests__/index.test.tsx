/**
 * @jest-environment jsdom
 */

import React from 'react';
import {
  moonDesignDark,
  moonDesignLight,
  ThemeProvider,
} from '@heathmont/moon-themes';
import renderer, { act } from 'react-test-renderer';
import Background from '../Background';
import Default from '../Default';
import ContentOutside from '../ContentOutside';
import ContentHeader from '../ContentHeader';
import Sizes from '../Sizes';
import WithoutButton from '../WithoutButton';
import ContentOutsideSizes from '../ContentOutsideSizes';
import OpenDefault from '../OpenDefault';
import Disabled from '../Disabled';

const renderWithTheme = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignLight}>{component}</ThemeProvider>
);

const renderWithThemeRTL = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignLight}>
    <div dir="rtl">{component}</div>
  </ThemeProvider>
);

const renderWithDarkTheme = (component: JSX.Element) => (
  <ThemeProvider theme={moonDesignDark}>{component}</ThemeProvider>
);

describe('Accordion', () => {
  it('renders default', () => {
    const testRenderer = renderer.create(renderWithTheme(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders default rtl', () => {
    const testRenderer = renderer.create(renderWithThemeRTL(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders default dark', () => {
    const testRenderer = renderer.create(renderWithDarkTheme(<Default />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders opened by default', () => {
    const testRenderer = renderer.create(renderWithTheme(<OpenDefault />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders without button', () => {
    const testRenderer = renderer.create(renderWithTheme(<WithoutButton />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders disabled', () => {
    const testRenderer = renderer.create(renderWithTheme(<Disabled />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders content outside', () => {
    const testRenderer = renderer.create(renderWithTheme(<ContentOutside />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders header content', () => {
    const testRenderer = renderer.create(renderWithTheme(<ContentHeader />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders sizes', () => {
    const testRenderer = renderer.create(renderWithTheme(<Sizes />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders sizes (content outside)', () => {
    const testRenderer = renderer.create(
      renderWithTheme(<ContentOutsideSizes />)
    );
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('expands', () => {
    const testRenderer = renderer.create(renderWithTheme(<Default />));
    const testInstance = testRenderer.root;
    act(() => {
      const button = testInstance.findByType('button');
      button && button?.props?.onClick();
    });
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });

  it('renders with background', () => {
    const testRenderer = renderer.create(renderWithTheme(<Background />));
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
