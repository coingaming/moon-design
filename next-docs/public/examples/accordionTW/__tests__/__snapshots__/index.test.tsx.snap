// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion New expands 1`] = `
<div
  className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-2 ps-3 pe-2"
>
  <div
    className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
    onClick={[Function]}
  >
    <h3
      className="flex-1 font-medium text-bulma text-moon-14"
    >
      Default
    </h3>
    <button
      className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
      type="button"
    >
      <svg
        className="text-trunks text-moon-24 transition-transform transition-200 rotate-90"
        fill="none"
        height="1em"
        viewBox="0 0 32 32"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 7l9 9-9 9"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </button>
  </div>
  <div
    className="overflow-hidden w-full text-bulma h-full mt-2"
  >
    <div
      className="text-moon-14"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
  </div>
</div>
`;

exports[`Accordion New renders 1`] = `
<div
  className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-2 ps-3 pe-2"
>
  <div
    className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
    onClick={[Function]}
  >
    <h3
      className="flex-1 font-medium text-bulma text-moon-14"
    >
      Default
    </h3>
    <button
      className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
      type="button"
    >
      <svg
        className="text-trunks text-moon-24 transition-transform transition-200"
        fill="none"
        height="1em"
        viewBox="0 0 32 32"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 7l9 9-9 9"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </button>
  </div>
  <div
    className="overflow-hidden w-full text-bulma h-0"
  >
    <div
      className="text-moon-14"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
  </div>
</div>
`;

exports[`Accordion New renders RTL 1`] = `
<div
  dir="rtl"
>
  <div
    className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-2 ps-3 pe-2"
  >
    <div
      className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
      onClick={[Function]}
    >
      <h3
        className="flex-1 font-medium text-bulma text-moon-14"
      >
        Default
      </h3>
      <button
        className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
        type="button"
      >
        <svg
          className="text-trunks text-moon-24 transition-transform transition-200"
          fill="none"
          height="1em"
          viewBox="0 0 32 32"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.5 7l9 9-9 9"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
      </button>
    </div>
    <div
      className="overflow-hidden w-full text-bulma h-0"
    >
      <div
        className="text-moon-14"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </div>
    </div>
  </div>
</div>
`;

exports[`Accordion New renders RTL 2`] = `
<div
  className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-2 ps-3 pe-2"
>
  <div
    className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
    onClick={[Function]}
  >
    <h3
      className="flex-1 font-medium text-bulma text-moon-14"
    >
      RTL Accordion
    </h3>
    <button
      className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
      type="button"
    >
      <svg
        className="text-trunks text-moon-24 transition-transform transition-200 rotate-180"
        fill="none"
        height="1em"
        viewBox="0 0 32 32"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 7l9 9-9 9"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </button>
  </div>
  <div
    className="overflow-hidden w-full text-bulma h-0"
  >
    <div
      className="text-moon-14"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
  </div>
</div>
`;

exports[`Accordion New renders content outside 1`] = `
<div
  className="w-full rounded-moon-s-sm h-max flex flex-col items-center transparent"
>
  <div
    className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer transparent rounded-moon-s-sm py-2 ps-3 pe-2"
    onClick={[Function]}
  >
    <h3
      className="flex-1 font-medium text-bulma text-moon-14"
    >
      Test accordion
    </h3>
    <button
      className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
      type="button"
    >
      <svg
        className="text-trunks text-moon-24 transition-transform transition-200"
        fill="none"
        height="1em"
        viewBox="0 0 32 32"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 7l9 9-9 9"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </button>
  </div>
  <div
    className="overflow-hidden w-full text-bulma h-0"
  >
    <div
      className="text-moon-14"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
  </div>
</div>
`;

exports[`Accordion New renders sizes variants 1`] = `
Array [
  <div
    className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan p-4"
  >
    <div
      className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
      onClick={[Function]}
    >
      <h3
        className="flex-1 font-medium text-bulma text-moon-16"
      >
        X Large (Default)
      </h3>
      <button
        className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
        type="button"
      >
        <svg
          className="text-trunks text-moon-24 transition-transform transition-200"
          fill="none"
          height="1em"
          viewBox="0 0 32 32"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.5 7l9 9-9 9"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
      </button>
    </div>
    <div
      className="overflow-hidden w-full text-bulma h-0"
    >
      <div
        className="text-moon-14"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </div>
    </div>
  </div>,
  <div
    className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan p-3"
  >
    <div
      className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
      onClick={[Function]}
    >
      <h3
        className="flex-1 font-medium text-bulma text-moon-14"
      >
        Large (lg)
      </h3>
      <button
        className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
        type="button"
      >
        <svg
          className="text-trunks text-moon-24 transition-transform transition-200"
          fill="none"
          height="1em"
          viewBox="0 0 32 32"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.5 7l9 9-9 9"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
      </button>
    </div>
    <div
      className="overflow-hidden w-full text-bulma h-0"
    >
      <div
        className="text-moon-14"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </div>
    </div>
  </div>,
  <div
    className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-2 ps-3 pe-2"
  >
    <div
      className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
      onClick={[Function]}
    >
      <h3
        className="flex-1 font-medium text-bulma text-moon-14"
      >
        Medium is default (md)
      </h3>
      <button
        className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
        type="button"
      >
        <svg
          className="text-trunks text-moon-24 transition-transform transition-200"
          fill="none"
          height="1em"
          viewBox="0 0 32 32"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.5 7l9 9-9 9"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
      </button>
    </div>
    <div
      className="overflow-hidden w-full text-bulma h-0"
    >
      <div
        className="text-moon-14"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </div>
    </div>
  </div>,
  <div
    className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-1 ps-3 pe-1"
  >
    <div
      className="bg-gohan w-full flex items-center relative gap-1 cursor-pointer bg-gohan"
      onClick={[Function]}
    >
      <h3
        className="flex-1 font-medium text-bulma text-moon-14"
      >
        Small (sm)
      </h3>
      <button
        className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-pointer"
        type="button"
      >
        <svg
          className="text-trunks text-moon-24 transition-transform transition-200"
          fill="none"
          height="1em"
          viewBox="0 0 32 32"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.5 7l9 9-9 9"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
          />
        </svg>
      </button>
    </div>
    <div
      className="overflow-hidden w-full text-bulma h-0"
    >
      <div
        className="text-moon-14"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </div>
    </div>
  </div>,
]
`;

exports[`Accordion New renders without button 1`] = `
<div
  className="w-full rounded-moon-s-sm h-max flex flex-col items-center bg-gohan py-2 ps-3 pe-2"
>
  <div
    className="bg-gohan w-full flex items-center relative gap-1 cursor-not-allowed bg-gohan"
    onClick={[Function]}
  >
    <h3
      className="flex-1 font-medium text-bulma text-moon-14"
    >
      Test accordion
    </h3>
    <button
      className="align-middle text-[0.5rem] leading-none no-underline text-trunks border-transparent bg-gohan cursor-not-allowed"
      disabled={true}
      type="button"
    >
      <svg
        className="text-trunks text-moon-24 transition-transform transition-200"
        fill="none"
        height="1em"
        viewBox="0 0 32 32"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M11.5 7l9 9-9 9"
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </button>
  </div>
  <div
    className="overflow-hidden w-full text-bulma h-0"
  >
    <div
      className="text-moon-14"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
  </div>
</div>
`;
